wormhole-connect/src/views/v2/Bridge/AssetPicker/TokenList.tsx

```diff
@@ -162,15 +166,25 @@ const TokenList = (props: Props) => {
     }
 
     return tokens;
-  }, [balances, props.tokenList, props.sourceToken]);
+  }, [
+    balances,
```

Comment:
This deps list was super outdated. Most of them are not expected to change while the token list is open (such as the connected wallet) but I still prefer to have them here.